docs/book/src/SUMMARY.md

```diff
@@ -39,7 +39,7 @@
   - [Hubs, spokes, and other wheel metaphors](./multiversion-tutorial/conversion-concepts.md)
   - [Implementing conversion](./multiversion-tutorial/conversion.md)
 
-      - [and setting up the webhooks](./multiversion-tutorial/webhooks.md)
+    - [and setting up the webhooks](./multiversion-tutorial/webhooks.md)
```

Comment:
We need revert this changes since it change the numbers/layout of the docs for no related items